src/www/ui/core-schema.dat

```diff
@@ -1983,6 +1987,7 @@
   $Schema["CONSTRAINT"]["pfile_pkey"] = "ALTER TABLE \"pfile\" ADD CONSTRAINT \"pfile_pkey\" PRIMARY KEY (\"pfile_pk\");";
   $Schema["CONSTRAINT"]["pfile_md5_sha1_size_ukey"] = "ALTER TABLE \"pfile\" ADD CONSTRAINT \"pfile_md5_sha1_size_ukey\" UNIQUE (\"pfile_md5\",\"pfile_sha1\",\"pfile_size\");";
   $Schema["CONSTRAINT"]["pfile_mimetype_fk_fkey"] = "ALTER TABLE \"pfile\" ADD CONSTRAINT \"pfile_mimetype_fk_fkey\" FOREIGN KEY (\"pfile_mimetypefk\") REFERENCES \"mimetype\" (\"mimetype_pk\") ON UPDATE NO ACTION ON DELETE NO ACTION;";
+  $Schema["CONSTRAINT"]["pfile_sha256_ukey"] = "ALTER TABLE \"pfile\" ADD CONSTRAINT \"pfile_sha256_ukey\" UNIQUE (\"pfile_sha256\");";  
```

Comment:
This constraint is not required. Instead, please update the constraint `pfile_md5_sha1_size_ukey` to include the new `pfile_sha256` column in the constraint.